### ISSUE TITLE
Looking for sgemm_ - not found

### ISSUE THREAD

**Issue by Marie0909** [alecjacobson/gptoolbox#143]
-- Looking for sgemm_
-- Looking for sgemm_ - not found
CMake Error at C:/Program Files/CMake/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find BLAS (missing: BLAS_LIBRARIES)
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files/CMake/share/cmake-3.23/Modules/FindBLAS.cmake:1337 (find_package_handle_standard_args)
  CMakeLists.txt:81 (find_package)

-- Configuring incomplete, errors occurred!
See also "E:/2022_ECG_inference/Cobiveco_IBME/dependencies/gptoolbox/mex/build/CMakeFiles/CMakeOutput.log".
See also "E:/2022_ECG_inference/Cobiveco_IBME/dependencies/gptoolbox/mex/build/CMakeFiles/CMakeError.log".

**Comment by alecjacobson** [comment]
Looks like this is on windows. Could you give a bit more information about your setup?

**Comment by Marie0909** [comment]
Yes, it is on Windows. Not quite sure what kinds of setup you mean:
It is on windows10, with VS code 2022, Matlab 2021b, VTK 9.1.0.

**Comment by alecjacobson** [comment]
I don't remember the situation for Windows. I think either you should install openblas or it should have been finding blas through matlab. Ideally it finds blas through matlab. If that's not the case then maybe our cmake file should take care of openblas.

I'm not a windows user so this is difficult for me to debug. Perhaps @otmanon or @rarora7777 may remember/know.

**Comment by ZZT-console** [comment]
I had solved it. You need to set compiler of cmake MinGW64, Then, download openblas library. Finally, copy its lib and include of openblas to lib and include of MinGW!
You try the following commend again:
cmake -G "MinGW Makefiles" ..

**Comment by JingyiT** [comment]
> I had solved it. You need to set compiler of cmake MinGW64, Then, download openblas library. Finally, copy its lib and include of openblas to lib and include of MinGW! You try the following commend again: cmake -G "MinGW Makefiles" ..

Thanks a lot! Your solution really helps!
